.github/workflows/hugegraph-llm.yml

```
@@ -0,0 +1,83 @@
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.10", "3.11"]
+
+    steps:
+    - name: Prepare HugeGraph Server Environment
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)